fluent-react/src/localized.js

```diff
@@ -116,6 +117,10 @@ export default class Localized extends Component {
       }
     }
 
+    if (!elem) {
```

Comment:
Given my comment above, I think it would be better to test `!isValidElement(elem)`. You can also move this up a bit, above the logic which translates attributes which doesn't apply for strings.

Reply:
Ok, that makes sense -- I've changed this to use just the message if children isn't a valid react element as suggested.